search/reverse-geocode-online/README.md

```diff
@@ -1,31 +1,30 @@
-# Reverse Geocode Online
+# Reverse geocode online
 
-Find the address of a location.
+Use an online service to find the address for a tapped point.
```

Comment:
tapped is very platform specific, surprised this made it through common samples approval
```suggestion
Use an online service to find the address for a point on the map.
```

Reply:
Good catch!